release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -749,13 +749,18 @@ In the following tables, features are marked with the following statuses:
 |Admin Network Policy (`AdminNetworkPolicy`)
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
```

Comment:
LGTM from ANP side.
@JoeAldinger will do another PR for the paragraph thingy

Reply:
done